{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "molecular-stylus",
   "metadata": {},
   "source": [
    "### UNSUPERVISED MACHINE LEARNING FOR THE CLASSIFICATION OF ASTROPHYSICAL X-RAY SOURCES"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "intimate-shift",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "\n",
    "from astroquery.simbad import Simbad\n",
    "import astropy.coordinates as coord\n",
    "import astropy.units as u\n",
    "\n",
    "Simbad.TIMEOUT = 60000 # sets the timeout to 60s"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "controlled-begin",
   "metadata": {},
   "outputs": [],
   "source": [
    "data = pd.read_csv(\"./cluster_data/astrox_knn6_src.csv\", index_col=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "convertible-television",
   "metadata": {},
   "outputs": [
    {
     "output_type": "execute_result",
     "data": {
      "text/plain": [
       "       cluster                   name          ra        dec  src_area_b  \\\n",
       "0            2  2CXO J000002.9-350332    0.012318 -35.059068    6.228827   \n",
       "1            2  2CXO J000010.0-501526    0.041803 -50.257400    8.978497   \n",
       "2            2  2CXO J000019.8-245030    0.082814 -24.841752  884.616067   \n",
       "3            0  2CXO J000025.4-245419    0.106246 -24.905300   39.749528   \n",
       "4            0  2CXO J000027.4-500421    0.114303 -50.072669    3.925662   \n",
       "...        ...                    ...         ...        ...         ...   \n",
       "37872        2  2CXO J235928.0+440317  359.866965  44.054832   32.554951   \n",
       "37873        0  2CXO J235932.4+181247  359.885089  18.213181    0.014842   \n",
       "37874        5  2CXO J235937.1+623151  359.904697  62.530855    0.156711   \n",
       "37875        2  2CXO J235945.8-574927  359.940857 -57.824258    6.741334   \n",
       "37877        2  2CXO J235959.4-573811  359.997656 -57.636390   72.362612   \n",
       "\n",
       "        hard_hm   hard_hs   hard_ms  var_prob_b  var_sigma_b  ...  \\\n",
       "0     -0.042473 -0.440974 -0.405996    0.067012     0.000041  ...   \n",
       "1     -0.049969 -0.277327 -0.229856    0.060476     0.000008  ...   \n",
       "2      0.041224 -0.278576 -0.311056    0.092881     0.000060  ...   \n",
       "3      0.033729 -0.329794 -0.358526    0.327994     0.000311  ...   \n",
       "4      0.154903 -0.126171 -0.276077    0.041535     0.000013  ...   \n",
       "...         ...       ...       ...         ...          ...  ...   \n",
       "37872 -0.161149 -0.440974 -0.298563    0.348135     0.000966  ...   \n",
       "37873 -0.187383 -0.608370 -0.474703    0.193192     0.000232  ...   \n",
       "37874  0.193629  0.296065  0.104934    0.069176     0.000010  ...   \n",
       "37875  0.084947 -0.262336 -0.338538    0.065255     0.000026  ...   \n",
       "37877  0.163648 -0.049969 -0.212367    0.099865     0.000093  ...   \n",
       "\n",
       "       var_sigma_s  ks_prob_b  ks_prob_h  ks_prob_m  ks_prob_s  kp_prob_b  \\\n",
       "0         0.000059   0.491601   0.724755   0.317516   0.423297   0.808309   \n",
       "1         0.000093   0.441207   0.249246   0.181138   0.805236   0.614278   \n",
       "2         0.000058   0.798917   0.847319   0.495847   0.506963   0.430789   \n",
       "3         0.000019   0.932091   0.067084   0.939628   0.368044   0.924866   \n",
       "4         0.000006   0.816880   0.969048   0.227237   0.030124   0.766089   \n",
       "...            ...        ...        ...        ...        ...        ...   \n",
       "37872     0.000286   0.981183   0.906643   0.336316   0.967217   0.884821   \n",
       "37873     0.001025   0.934161   0.583490   0.964677   0.262496   0.974790   \n",
       "37874     0.000032   0.249008   0.052800   0.338171   0.093817   0.447750   \n",
       "37875     0.000149   0.701624   0.691945   0.910468   0.948922   0.882940   \n",
       "37877     0.000498   0.627601   0.480334   0.353657   0.987674   0.577222   \n",
       "\n",
       "       kp_prob_h  kp_prob_m  kp_prob_s     bb_kt  \n",
       "0       0.669796   0.197852   0.691868  0.439048  \n",
       "1       0.101385   0.028956   0.918830  0.534998  \n",
       "2       0.817322   0.353208   0.778730  0.414136  \n",
       "3       0.051374   0.950423   0.288221  0.456196  \n",
       "4       0.932166   0.556750   0.047099  0.628996  \n",
       "...          ...        ...        ...       ...  \n",
       "37872   0.704248   0.628147   0.834831  0.431097  \n",
       "37873   0.494843   0.989638   0.387899  0.288484  \n",
       "37874   0.109237   0.512777   0.100806  0.744936  \n",
       "37875   0.727829   0.955295   0.743982  0.618062  \n",
       "37877   0.595085   0.148076   0.970533  0.644100  \n",
       "\n",
       "[23025 rows x 25 columns]"
      ],
      "text/html": "<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>cluster</th>\n      <th>name</th>\n      <th>ra</th>\n      <th>dec</th>\n      <th>src_area_b</th>\n      <th>hard_hm</th>\n      <th>hard_hs</th>\n      <th>hard_ms</th>\n      <th>var_prob_b</th>\n      <th>var_sigma_b</th>\n      <th>...</th>\n      <th>var_sigma_s</th>\n      <th>ks_prob_b</th>\n      <th>ks_prob_h</th>\n      <th>ks_prob_m</th>\n      <th>ks_prob_s</th>\n      <th>kp_prob_b</th>\n      <th>kp_prob_h</th>\n      <th>kp_prob_m</th>\n      <th>kp_prob_s</th>\n      <th>bb_kt</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>2</td>\n      <td>2CXO J000002.9-350332</td>\n      <td>0.012318</td>\n      <td>-35.059068</td>\n      <td>6.228827</td>\n      <td>-0.042473</td>\n      <td>-0.440974</td>\n      <td>-0.405996</td>\n      <td>0.067012</td>\n      <td>0.000041</td>\n      <td>...</td>\n      <td>0.000059</td>\n      <td>0.491601</td>\n      <td>0.724755</td>\n      <td>0.317516</td>\n      <td>0.423297</td>\n      <td>0.808309</td>\n      <td>0.669796</td>\n      <td>0.197852</td>\n      <td>0.691868</td>\n      <td>0.439048</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>2</td>\n      <td>2CXO J000010.0-501526</td>\n      <td>0.041803</td>\n      <td>-50.257400</td>\n      <td>8.978497</td>\n      <td>-0.049969</td>\n      <td>-0.277327</td>\n      <td>-0.229856</td>\n      <td>0.060476</td>\n      <td>0.000008</td>\n      <td>...</td>\n      <td>0.000093</td>\n      <td>0.441207</td>\n      <td>0.249246</td>\n      <td>0.181138</td>\n      <td>0.805236</td>\n      <td>0.614278</td>\n      <td>0.101385</td>\n      <td>0.028956</td>\n      <td>0.918830</td>\n      <td>0.534998</td>\n    </tr>\n    <tr>\n      <th>2</th>\n      <td>2</td>\n      <td>2CXO J000019.8-245030</td>\n      <td>0.082814</td>\n      <td>-24.841752</td>\n      <td>884.616067</td>\n      <td>0.041224</td>\n      <td>-0.278576</td>\n      <td>-0.311056</td>\n      <td>0.092881</td>\n      <td>0.000060</td>\n      <td>...</td>\n      <td>0.000058</td>\n      <td>0.798917</td>\n      <td>0.847319</td>\n      <td>0.495847</td>\n      <td>0.506963</td>\n      <td>0.430789</td>\n      <td>0.817322</td>\n      <td>0.353208</td>\n      <td>0.778730</td>\n      <td>0.414136</td>\n    </tr>\n    <tr>\n      <th>3</th>\n      <td>0</td>\n      <td>2CXO J000025.4-245419</td>\n      <td>0.106246</td>\n      <td>-24.905300</td>\n      <td>39.749528</td>\n      <td>0.033729</td>\n      <td>-0.329794</td>\n      <td>-0.358526</td>\n      <td>0.327994</td>\n      <td>0.000311</td>\n      <td>...</td>\n      <td>0.000019</td>\n      <td>0.932091</td>\n      <td>0.067084</td>\n      <td>0.939628</td>\n      <td>0.368044</td>\n      <td>0.924866</td>\n      <td>0.051374</td>\n      <td>0.950423</td>\n      <td>0.288221</td>\n      <td>0.456196</td>\n    </tr>\n    <tr>\n      <th>4</th>\n      <td>0</td>\n      <td>2CXO J000027.4-500421</td>\n      <td>0.114303</td>\n      <td>-50.072669</td>\n      <td>3.925662</td>\n      <td>0.154903</td>\n      <td>-0.126171</td>\n      <td>-0.276077</td>\n      <td>0.041535</td>\n      <td>0.000013</td>\n      <td>...</td>\n      <td>0.000006</td>\n      <td>0.816880</td>\n      <td>0.969048</td>\n      <td>0.227237</td>\n      <td>0.030124</td>\n      <td>0.766089</td>\n      <td>0.932166</td>\n      <td>0.556750</td>\n      <td>0.047099</td>\n      <td>0.628996</td>\n    </tr>\n    <tr>\n      <th>...</th>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n      <td>...</td>\n    </tr>\n    <tr>\n      <th>37872</th>\n      <td>2</td>\n      <td>2CXO J235928.0+440317</td>\n      <td>359.866965</td>\n      <td>44.054832</td>\n      <td>32.554951</td>\n      <td>-0.161149</td>\n      <td>-0.440974</td>\n      <td>-0.298563</td>\n      <td>0.348135</td>\n      <td>0.000966</td>\n      <td>...</td>\n      <td>0.000286</td>\n      <td>0.981183</td>\n      <td>0.906643</td>\n      <td>0.336316</td>\n      <td>0.967217</td>\n      <td>0.884821</td>\n      <td>0.704248</td>\n      <td>0.628147</td>\n      <td>0.834831</td>\n      <td>0.431097</td>\n    </tr>\n    <tr>\n      <th>37873</th>\n      <td>0</td>\n      <td>2CXO J235932.4+181247</td>\n      <td>359.885089</td>\n      <td>18.213181</td>\n      <td>0.014842</td>\n      <td>-0.187383</td>\n      <td>-0.608370</td>\n      <td>-0.474703</td>\n      <td>0.193192</td>\n      <td>0.000232</td>\n      <td>...</td>\n      <td>0.001025</td>\n      <td>0.934161</td>\n      <td>0.583490</td>\n      <td>0.964677</td>\n      <td>0.262496</td>\n      <td>0.974790</td>\n      <td>0.494843</td>\n      <td>0.989638</td>\n      <td>0.387899</td>\n      <td>0.288484</td>\n    </tr>\n    <tr>\n      <th>37874</th>\n      <td>5</td>\n      <td>2CXO J235937.1+623151</td>\n      <td>359.904697</td>\n      <td>62.530855</td>\n      <td>0.156711</td>\n      <td>0.193629</td>\n      <td>0.296065</td>\n      <td>0.104934</td>\n      <td>0.069176</td>\n      <td>0.000010</td>\n      <td>...</td>\n      <td>0.000032</td>\n      <td>0.249008</td>\n      <td>0.052800</td>\n      <td>0.338171</td>\n      <td>0.093817</td>\n      <td>0.447750</td>\n      <td>0.109237</td>\n      <td>0.512777</td>\n      <td>0.100806</td>\n      <td>0.744936</td>\n    </tr>\n    <tr>\n      <th>37875</th>\n      <td>2</td>\n      <td>2CXO J235945.8-574927</td>\n      <td>359.940857</td>\n      <td>-57.824258</td>\n      <td>6.741334</td>\n      <td>0.084947</td>\n      <td>-0.262336</td>\n      <td>-0.338538</td>\n      <td>0.065255</td>\n      <td>0.000026</td>\n      <td>...</td>\n      <td>0.000149</td>\n      <td>0.701624</td>\n      <td>0.691945</td>\n      <td>0.910468</td>\n      <td>0.948922</td>\n      <td>0.882940</td>\n      <td>0.727829</td>\n      <td>0.955295</td>\n      <td>0.743982</td>\n      <td>0.618062</td>\n    </tr>\n    <tr>\n      <th>37877</th>\n      <td>2</td>\n      <td>2CXO J235959.4-573811</td>\n      <td>359.997656</td>\n      <td>-57.636390</td>\n      <td>72.362612</td>\n      <td>0.163648</td>\n      <td>-0.049969</td>\n      <td>-0.212367</td>\n      <td>0.099865</td>\n      <td>0.000093</td>\n      <td>...</td>\n      <td>0.000498</td>\n      <td>0.627601</td>\n      <td>0.480334</td>\n      <td>0.353657</td>\n      <td>0.987674</td>\n      <td>0.577222</td>\n      <td>0.595085</td>\n      <td>0.148076</td>\n      <td>0.970533</td>\n      <td>0.644100</td>\n    </tr>\n  </tbody>\n</table>\n<p>23025 rows × 25 columns</p>\n</div>"
     },
     "metadata": {},
     "execution_count": 3
    }
   ],
   "source": [
    "data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "consistent-drinking",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_0 = data[data.cluster == 0]\n",
    "data_1 = data[data.cluster == 1]\n",
    "data_2 = data[data.cluster == 2]\n",
    "data_3 = data[data.cluster == 3]\n",
    "data_4 = data[data.cluster == 4]\n",
    "data_5 = data[data.cluster == 5]\n",
    "\n",
    "n = 500\n",
    "data_0_sample = data_0\n",
    "data_1_sample = data_1\n",
    "data_2_sample = data_2\n",
    "data_3_sample = data_3\n",
    "data_4_sample = data_4\n",
    "data_5_sample = data_5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "physical-legislation",
   "metadata": {},
   "outputs": [],
   "source": [
    "Simbad.add_votable_fields('typed_id')\n",
    "Simbad.add_votable_fields('otype')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "forced-newton",
   "metadata": {
    "tags": [
     "outputPrepend"
    ]
   },
   "outputs": [
    {
     "output_type": "stream",
     "name": "stderr",
     "text": [
      "result table): '21:07:27.6373 -58:56:46.9936': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4061 raised an error (recorded in the `errors` attribute of the result table): '21:10:12.6521 +21:25:50.5536': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4063 raised an error (recorded in the `errors` attribute of the result table): '21:12:45.6186 -63:23:29.5734': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4066 raised an error (recorded in the `errors` attribute of the result table): '21:24:29.6235 -34:07:08.2948': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4071 raised an error (recorded in the `errors` attribute of the result table): '21:36:19.9241 -54:31:46.7897': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4076 raised an error (recorded in the `errors` attribute of the result table): '21:39:49.5049 +12:09:14.8143': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4079 raised an error (recorded in the `errors` attribute of the result table): '21:40:41.1058 -23:13:44.5095': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4081 raised an error (recorded in the `errors` attribute of the result table): '21:43:59.4762 +19:17:53.8592': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4083 raised an error (recorded in the `errors` attribute of the result table): '21:46:47.7553 -46:42:01.659': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4085 raised an error (recorded in the `errors` attribute of the result table): '21:46:57.0632 -46:42:23.603': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4089 raised an error (recorded in the `errors` attribute of the result table): '21:51:43.7198 -5:36:42.9378': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4091 raised an error (recorded in the `errors` attribute of the result table): '21:53:19.1903 +17:42:04.3939': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4096 raised an error (recorded in the `errors` attribute of the result table): '21:55:17.8272 +8:50:14.4027': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4098 raised an error (recorded in the `errors` attribute of the result table): '21:55:40.6629 +37:49:28.6386': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4100 raised an error (recorded in the `errors` attribute of the result table): '21:56:07.6944 +8:42:37.0931': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4102 raised an error (recorded in the `errors` attribute of the result table): '22:00:11.259 -39:28:53.1314': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4105 raised an error (recorded in the `errors` attribute of the result table): '22:14:10.7568 -10:17:20.5814': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4107 raised an error (recorded in the `errors` attribute of the result table): '22:14:50.0256 +13:55:19.3725': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4109 raised an error (recorded in the `errors` attribute of the result table): '22:15:31.9678 -17:00:58.693': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4111 raised an error (recorded in the `errors` attribute of the result table): '22:15:41.6937 -17:24:57.6567': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4119 raised an error (recorded in the `errors` attribute of the result table): '22:22:29.4869 -48:34:57.6563': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4121 raised an error (recorded in the `errors` attribute of the result table): '22:24:03.3694 -2:06:00.8396': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4123 raised an error (recorded in the `errors` attribute of the result table): '22:26:35.9651 +65:35:37.5566': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4126 raised an error (recorded in the `errors` attribute of the result table): '22:28:46.9846 -13:15:01.3723': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4132 raised an error (recorded in the `errors` attribute of the result table): '22:35:55.1051 -25:58:34.0623': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4134 raised an error (recorded in the `errors` attribute of the result table): '22:36:04.1819 +34:12:11.1277': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4137 raised an error (recorded in the `errors` attribute of the result table): '22:38:31.3499 -17:31:17.2917': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4139 raised an error (recorded in the `errors` attribute of the result table): '22:39:11.2167 -40:14:00.6607': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4141 raised an error (recorded in the `errors` attribute of the result table): '22:39:12.5988 -5:43:23.1467': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4143 raised an error (recorded in the `errors` attribute of the result table): '22:39:39.518 -40:15:41.1211': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4145 raised an error (recorded in the `errors` attribute of the result table): '22:40:19.2085 +3:26:13.9118': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4149 raised an error (recorded in the `errors` attribute of the result table): '22:40:30.1721 +3:21:31.8821': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4154 raised an error (recorded in the `errors` attribute of the result table): '22:46:03.7355 +39:53:32.4991': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4157 raised an error (recorded in the `errors` attribute of the result table): '22:49:32.1364 +11:31:31.4791': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4161 raised an error (recorded in the `errors` attribute of the result table): '22:54:57.1848 +62:34:16.1772': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4178 raised an error (recorded in the `errors` attribute of the result table): '22:57:06.8454 +62:38:29.1397': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4183 raised an error (recorded in the `errors` attribute of the result table): '22:58:46.6357 -34:40:14.769': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4185 raised an error (recorded in the `errors` attribute of the result table): '22:58:57.1823 -34:58:12.8926': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4187 raised an error (recorded in the `errors` attribute of the result table): '22:59:03.6506 -60:59:29.6036': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4189 raised an error (recorded in the `errors` attribute of the result table): '23:02:04.3634 -40:32:14.821': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4192 raised an error (recorded in the `errors` attribute of the result table): '23:02:35.3306 -39:00:57.2626': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4194 raised an error (recorded in the `errors` attribute of the result table): '23:02:49.0927 +8:51:52.2574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4197 raised an error (recorded in the `errors` attribute of the result table): '23:07:44.0443 -9:54:37.246': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4199 raised an error (recorded in the `errors` attribute of the result table): '23:12:15.8903 -21:40:05.7416': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4201 raised an error (recorded in the `errors` attribute of the result table): '23:13:47.5124 -21:52:14.3907': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4203 raised an error (recorded in the `errors` attribute of the result table): '23:14:14.2208 +4:27:25.7104': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4205 raised an error (recorded in the `errors` attribute of the result table): '23:18:38.505 +42:52:59.8923': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4210 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4213 raised an error (recorded in the `errors` attribute of the result table): '23:24:52.9161 +16:52:38.77': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4216 raised an error (recorded in the `errors` attribute of the result table): '23:27:29.6246 +23:49:52.1604': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4219 raised an error (recorded in the `errors` attribute of the result table): '23:27:46.3158 -2:10:20.4485': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4221 raised an error (recorded in the `errors` attribute of the result table): '23:27:57.2478 -2:09:51.6564': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4225 raised an error (recorded in the `errors` attribute of the result table): '23:34:27.5596 -36:11:04.6313': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4228 raised an error (recorded in the `errors` attribute of the result table): '23:43:28.5387 -55:30:37.8281': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4230 raised an error (recorded in the `errors` attribute of the result table): '23:43:35.9424 -15:19:16.2326': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4233 raised an error (recorded in the `errors` attribute of the result table): '23:45:39.0858 -64:12:42.8833': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4237 raised an error (recorded in the `errors` attribute of the result table): '23:51:49.9653 -46:56:52.8728': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4239 raised an error (recorded in the `errors` attribute of the result table): '23:55:52.6891 +19:28:20.4976': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4242 raised an error (recorded in the `errors` attribute of the result table): '23:58:08.826 -32:34:03.6433': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4245 raised an error (recorded in the `errors` attribute of the result table): '23:59:25.6994 -50:07:54.3791': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\n"
     ]
    }
   ],
   "source": [
    "coords = coord.SkyCoord(data_0_sample.ra, data_0_sample.dec,unit=(u.deg, u.deg), frame='icrs')\n",
    "result_table = Simbad.query_region(coords, radius=0.000277778 * u.deg); # 36 arcsec\n",
    "resdf = result_table.to_pandas()\n",
    "count_obj = resdf.groupby(resdf.OTYPE.tolist(),as_index=False).size()\n",
    "count_obj.to_csv('./data/obj/0_count_obj{}.csv'.format(n))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "covered-investing",
   "metadata": {
    "tags": [
     "outputPrepend"
    ]
   },
   "outputs": [
    {
     "output_type": "stream",
     "name": "stderr",
     "text": [
      "t table): '18:03:48.4537 -24:19:40.0364': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2117 raised an error (recorded in the `errors` attribute of the result table): '18:03:49.4477 -24:23:30.7581': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2125 raised an error (recorded in the `errors` attribute of the result table): '18:03:54.3945 -24:23:54.3712': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2133 raised an error (recorded in the `errors` attribute of the result table): '18:04:01.1095 -24:26:03.0539': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2169 raised an error (recorded in the `errors` attribute of the result table): '18:04:37.0257 -24:29:59.7822': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2185 raised an error (recorded in the `errors` attribute of the result table): '18:09:46.7341 -23:38:45.8354': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2188 raised an error (recorded in the `errors` attribute of the result table): '18:11:13.644 -19:33:41.1405': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2190 raised an error (recorded in the `errors` attribute of the result table): '18:11:26.9145 -19:24:42.8377': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2192 raised an error (recorded in the `errors` attribute of the result table): '18:11:42.0404 -19:19:42.0369': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2194 raised an error (recorded in the `errors` attribute of the result table): '18:12:00.7689 -19:19:15.8337': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2196 raised an error (recorded in the `errors` attribute of the result table): '18:17:06.8238 -16:04:45.4998': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2198 raised an error (recorded in the `errors` attribute of the result table): '18:18:06.1627 -16:53:50.3878': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2211 raised an error (recorded in the `errors` attribute of the result table): '18:19:40.2894 -13:51:31.4481': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2217 raised an error (recorded in the `errors` attribute of the result table): '18:21:32.2577 -16:10:28.3524': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2220 raised an error (recorded in the `errors` attribute of the result table): '18:22:19.0804 -16:10:16.161': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2223 raised an error (recorded in the `errors` attribute of the result table): '18:24:31.9664 -24:39:55.0346': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2225 raised an error (recorded in the `errors` attribute of the result table): '18:24:49.3989 -24:52:07.8969': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2231 raised an error (recorded in the `errors` attribute of the result table): '18:29:34.5445 -1:58:00.1588': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2233 raised an error (recorded in the `errors` attribute of the result table): '18:29:37.6009 +1:03:24.6014': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2238 raised an error (recorded in the `errors` attribute of the result table): '18:35:22.5365 +33:04:12.5557': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2243 raised an error (recorded in the `errors` attribute of the result table): '18:40:25.8006 -5:37:16.6462': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2245 raised an error (recorded in the `errors` attribute of the result table): '18:42:49.7296 -32:12:47.1274': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2247 raised an error (recorded in the `errors` attribute of the result table): '18:48:56.2359 -1:23:43.6652': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2249 raised an error (recorded in the `errors` attribute of the result table): '18:48:58.7052 -0:50:27.3105': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2252 raised an error (recorded in the `errors` attribute of the result table): '18:57:27.4018 -49:05:31.8633': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2266 raised an error (recorded in the `errors` attribute of the result table): '19:04:35.722 +5:41:02.8743': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2268 raised an error (recorded in the `errors` attribute of the result table): '19:07:44.3654 +72:18:16.7361': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2274 raised an error (recorded in the `errors` attribute of the result table): '19:11:01.5416 +8:52:47.503': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2276 raised an error (recorded in the `errors` attribute of the result table): '19:15:27.5146 -24:18:26.4337': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2278 raised an error (recorded in the `errors` attribute of the result table): '19:15:33.1952 -24:07:57.2157': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2280 raised an error (recorded in the `errors` attribute of the result table): '19:16:49.6288 -16:13:35.8163': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2282 raised an error (recorded in the `errors` attribute of the result table): '19:17:43.3463 -16:06:35.4302': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2285 raised an error (recorded in the `errors` attribute of the result table): '19:23:01.7764 +13:59:52.492': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2288 raised an error (recorded in the `errors` attribute of the result table): '19:30:24.0246 +18:49:04.7964': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2291 raised an error (recorded in the `errors` attribute of the result table): '19:33:02.7035 -26:37:09.1055': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2293 raised an error (recorded in the `errors` attribute of the result table): '19:40:12.4748 +21:42:53.1603': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2296 raised an error (recorded in the `errors` attribute of the result table): '19:57:41.2358 -30:12:20.3552': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2299 raised an error (recorded in the `errors` attribute of the result table): '20:07:29.9587 -48:33:14.2403': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2302 raised an error (recorded in the `errors` attribute of the result table): '20:09:17.3154 -48:22:03.532': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2304 raised an error (recorded in the `errors` attribute of the result table): '20:12:08.7213 +38:26:12.7764': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2307 raised an error (recorded in the `errors` attribute of the result table): '20:18:10.5968 +36:30:47.7652': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2309 raised an error (recorded in the `errors` attribute of the result table): '20:20:16.9101 +40:26:24.2336': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2312 raised an error (recorded in the `errors` attribute of the result table): '20:20:43.7274 +40:27:00.3373': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2326 raised an error (recorded in the `errors` attribute of the result table): '20:33:46.0924 +60:14:19.1831': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2330 raised an error (recorded in the `errors` attribute of the result table): '20:33:59.5288 -59:43:17.2788': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2334 raised an error (recorded in the `errors` attribute of the result table): '20:35:52.9337 -35:38:20.8449': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2336 raised an error (recorded in the `errors` attribute of the result table): '20:47:14.6935 +11:04:41.6402': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2340 raised an error (recorded in the `errors` attribute of the result table): '21:10:58.1537 -63:32:45.2473': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2343 raised an error (recorded in the `errors` attribute of the result table): '21:35:23.1756 -54:20:08.5971': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2348 raised an error (recorded in the `errors` attribute of the result table): '21:44:03.8586 +19:28:12.598': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2351 raised an error (recorded in the `errors` attribute of the result table): '21:53:28.7692 +62:32:11.3193': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2353 raised an error (recorded in the `errors` attribute of the result table): '21:53:28.7692 +62:32:11.3193': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2359 raised an error (recorded in the `errors` attribute of the result table): '22:14:21.5874 -16:53:52.8121': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2364 raised an error (recorded in the `errors` attribute of the result table): '22:24:12.1203 +65:46:04.2514': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2369 raised an error (recorded in the `errors` attribute of the result table): '22:33:04.1989 -59:57:40.6012': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2373 raised an error (recorded in the `errors` attribute of the result table): '22:41:31.6908 +53:12:45.6124': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2382 raised an error (recorded in the `errors` attribute of the result table): '22:58:30.369 -40:42:44.6348': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2385 raised an error (recorded in the `errors` attribute of the result table): '23:02:43.9718 -40:19:32.017': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2389 raised an error (recorded in the `errors` attribute of the result table): '23:56:06.147 -34:50:55.3916': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 2391 raised an error (recorded in the `errors` attribute of the result table): '23:59:25.6994 -50:07:54.3791': No astronomical object found :\n  warnings.warn(\"Warning: The script line number %i raised \"\n"
     ]
    }
   ],
   "source": [
    "coords = coord.SkyCoord(data_1_sample.ra, data_1_sample.dec,unit=(u.deg, u.deg), frame='icrs')\n",
    "result_table = Simbad.query_region(coords, radius=0.000277778 * u.deg); # 36 arcsec\n",
    "resdf = result_table.to_pandas()\n",
    "count_obj = resdf.groupby(resdf.OTYPE.tolist(),as_index=False).size()\n",
    "count_obj.to_csv('./data/obj/1_count_obj{}.csv'.format(n))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "subtle-gregory",
   "metadata": {
    "tags": [
     "outputPrepend"
    ]
   },
   "outputs": [
    {
     "output_type": "stream",
     "name": "stderr",
     "text": [
      "result table): '21:47:58.5568 -61:29:09.3919': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4582 raised an error (recorded in the `errors` attribute of the result table): '21:53:28.7692 +62:32:11.3193': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4585 raised an error (recorded in the `errors` attribute of the result table): '21:54:23.1143 -57:45:56.9885': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4589 raised an error (recorded in the `errors` attribute of the result table): '21:55:04.3169 +38:08:08.5502': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4591 raised an error (recorded in the `errors` attribute of the result table): '21:55:50.0105 -4:41:48.4297': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4593 raised an error (recorded in the `errors` attribute of the result table): '21:56:29.5535 -69:39:05.7448': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4595 raised an error (recorded in the `errors` attribute of the result table): '21:57:09.7541 -4:52:10.0727': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4597 raised an error (recorded in the `errors` attribute of the result table): '21:59:19.702 -39:21:34.0843': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4601 raised an error (recorded in the `errors` attribute of the result table): '22:04:27.783 -56:44:16.3003': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4605 raised an error (recorded in the `errors` attribute of the result table): '22:14:08.6858 -17:01:00.7418': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4607 raised an error (recorded in the `errors` attribute of the result table): '22:14:16.7401 -17:01:46.1958': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4609 raised an error (recorded in the `errors` attribute of the result table): '22:14:39.3613 -14:00:58.8037': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4611 raised an error (recorded in the `errors` attribute of the result table): '22:14:39.5771 -17:14:28.6999': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4615 raised an error (recorded in the `errors` attribute of the result table): '22:16:51.8348 -17:25:49.2737': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4617 raised an error (recorded in the `errors` attribute of the result table): '22:16:57.2457 +0:24:09.6769': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4636 raised an error (recorded in the `errors` attribute of the result table): '22:25:23.5653 +17:16:34.7617': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4638 raised an error (recorded in the `errors` attribute of the result table): '22:25:30.9328 +17:25:37.1329': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4640 raised an error (recorded in the `errors` attribute of the result table): '22:26:35.9651 +65:35:37.5566': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4649 raised an error (recorded in the `errors` attribute of the result table): '22:36:21.7866 -25:57:43.5506': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4653 raised an error (recorded in the `errors` attribute of the result table): '22:37:58.4689 -45:54:52.2411': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4656 raised an error (recorded in the `errors` attribute of the result table): '22:39:04.3826 -5:39:21.7524': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4658 raised an error (recorded in the `errors` attribute of the result table): '22:40:05.0989 -40:15:23.052': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4666 raised an error (recorded in the `errors` attribute of the result table): '22:43:24.0125 +53:03:19.935': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4668 raised an error (recorded in the `errors` attribute of the result table): '22:44:55.321 +39:42:31.0457': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4670 raised an error (recorded in the `errors` attribute of the result table): '22:46:49.0729 -52:49:21.5609': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4673 raised an error (recorded in the `errors` attribute of the result table): '22:48:53.6089 -44:24:30.22': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4675 raised an error (recorded in the `errors` attribute of the result table): '22:53:53.9906 +62:35:47.4083': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4685 raised an error (recorded in the `errors` attribute of the result table): '22:56:20.325 +62:52:23.7063': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4694 raised an error (recorded in the `errors` attribute of the result table): '22:57:06.8454 +62:38:29.1397': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4699 raised an error (recorded in the `errors` attribute of the result table): '22:57:47.5969 -34:48:03.0091': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4701 raised an error (recorded in the `errors` attribute of the result table): '22:58:12.467 -41:13:01.8377': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4706 raised an error (recorded in the `errors` attribute of the result table): '22:59:08.7994 -34:59:11.0907': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4708 raised an error (recorded in the `errors` attribute of the result table): '23:01:11.3244 +58:47:47.2654': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4710 raised an error (recorded in the `errors` attribute of the result table): '23:01:41.4454 -40:07:53.5636': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4714 raised an error (recorded in the `errors` attribute of the result table): '23:03:00.7133 -40:24:25.8604': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4717 raised an error (recorded in the `errors` attribute of the result table): '23:07:22.7042 -22:40:34.6621': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4720 raised an error (recorded in the `errors` attribute of the result table): '23:11:26.3215 -21:47:52.4443': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4727 raised an error (recorded in the `errors` attribute of the result table): '23:18:35.5236 -4:30:25.1727': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4741 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4743 raised an error (recorded in the `errors` attribute of the result table): '23:23:54.8003 +58:50:00.5392': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4748 raised an error (recorded in the `errors` attribute of the result table): '23:25:31.2103 -12:10:35.7665': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4751 raised an error (recorded in the `errors` attribute of the result table): '23:26:48.5917 +42:39:11.324': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4754 raised an error (recorded in the `errors` attribute of the result table): '23:27:45.2134 +23:32:47.1255': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4758 raised an error (recorded in the `errors` attribute of the result table): '23:30:44.6004 +39:23:17.8031': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4761 raised an error (recorded in the `errors` attribute of the result table): '23:35:23.7849 -36:03:03.9721': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4764 raised an error (recorded in the `errors` attribute of the result table): '23:36:49.507 +21:16:45.64': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4767 raised an error (recorded in the `errors` attribute of the result table): '23:42:09.807 -51:20:05.025': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4769 raised an error (recorded in the `errors` attribute of the result table): '23:44:13.3859 -55:23:50.5718': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4771 raised an error (recorded in the `errors` attribute of the result table): '23:45:25.6518 -64:12:06.9198': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4773 raised an error (recorded in the `errors` attribute of the result table): '23:46:50.4463 -64:07:54.0674': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4775 raised an error (recorded in the `errors` attribute of the result table): '23:48:40.0796 +1:07:51.9455': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4777 raised an error (recorded in the `errors` attribute of the result table): '23:52:49.3298 -47:05:41.0126': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4779 raised an error (recorded in the `errors` attribute of the result table): '23:53:25.1407 -46:59:42.6606': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4781 raised an error (recorded in the `errors` attribute of the result table): '23:56:37.7027 -35:02:00.1391': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4784 raised an error (recorded in the `errors` attribute of the result table): '23:57:09.2722 -34:45:18.7978': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4786 raised an error (recorded in the `errors` attribute of the result table): '23:57:46.5886 -55:34:05.83': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4789 raised an error (recorded in the `errors` attribute of the result table): '23:58:21.142 -32:47:20.3376': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4791 raised an error (recorded in the `errors` attribute of the result table): '23:58:50.4966 -32:42:40.8667': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4793 raised an error (recorded in the `errors` attribute of the result table): '23:59:15.7862 -32:26:44.6253': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 4796 raised an error (recorded in the `errors` attribute of the result table): '23:59:45.8057 -57:49:27.3277': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\n"
     ]
    }
   ],
   "source": [
    "coords = coord.SkyCoord(data_2_sample.ra, data_2_sample.dec,unit=(u.deg, u.deg), frame='icrs')\n",
    "result_table = Simbad.query_region(coords, radius=0.000277778 * u.deg); # 36 arcsec # Cambiar a 10 arcsec o 5 arsec\n",
    "resdf = result_table.to_pandas()\n",
    "count_obj = resdf.groupby(resdf.OTYPE.tolist(),as_index=False).size()\n",
    "count_obj.to_csv('./data/obj/2_count_obj{}.csv'.format(n))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "herbal-dialogue",
   "metadata": {
    "tags": [
     "outputPrepend"
    ]
   },
   "outputs": [
    {
     "output_type": "stream",
     "name": "stderr",
     "text": [
      "lt table): '17:45:00.1384 -28:51:29.0518': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1463 raised an error (recorded in the `errors` attribute of the result table): '17:45:36.5153 -29:01:14.0426': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1469 raised an error (recorded in the `errors` attribute of the result table): '17:46:02.0453 -29:10:45.7769': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1479 raised an error (recorded in the `errors` attribute of the result table): '17:47:26.0209 -30:02:41.8844': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1481 raised an error (recorded in the `errors` attribute of the result table): '17:48:02.6507 -24:46:02.7416': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1483 raised an error (recorded in the `errors` attribute of the result table): '17:48:02.6507 -24:46:02.7416': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1491 raised an error (recorded in the `errors` attribute of the result table): '17:48:35.0247 -27:02:27.1754': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1496 raised an error (recorded in the `errors` attribute of the result table): '17:50:23.1017 -29:35:23.18': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1499 raised an error (recorded in the `errors` attribute of the result table): '17:51:38.3464 -29:42:40.428': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1501 raised an error (recorded in the `errors` attribute of the result table): '17:51:54.7124 -29:28:05.8642': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1504 raised an error (recorded in the `errors` attribute of the result table): '17:55:00.3355 -29:41:10.5008': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1506 raised an error (recorded in the `errors` attribute of the result table): '18:03:03.7397 -30:04:37.1295': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1519 raised an error (recorded in the `errors` attribute of the result table): '18:04:49.7296 -7:35:26.7993': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1524 raised an error (recorded in the `errors` attribute of the result table): '18:04:53.3339 -24:26:14.5977': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1528 raised an error (recorded in the `errors` attribute of the result table): '18:08:59.8529 -43:50:36.2914': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1530 raised an error (recorded in the `errors` attribute of the result table): '18:14:31.0733 -17:09:25.6979': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1532 raised an error (recorded in the `errors` attribute of the result table): '18:18:18.2249 -17:02:06.4175': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1544 raised an error (recorded in the `errors` attribute of the result table): '18:19:27.2467 -13:33:56.7002': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1547 raised an error (recorded in the `errors` attribute of the result table): '18:20:12.5883 -14:23:51.1457': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1578 raised an error (recorded in the `errors` attribute of the result table): '18:20:30.8188 -16:10:13.829': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1600 raised an error (recorded in the `errors` attribute of the result table): '18:21:00.4073 -16:10:36.107': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1602 raised an error (recorded in the `errors` attribute of the result table): '18:21:18.4946 -16:13:45.5951': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1612 raised an error (recorded in the `errors` attribute of the result table): '18:26:15.0565 -14:50:55.3329': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1614 raised an error (recorded in the `errors` attribute of the result table): '18:26:32.8139 -13:01:25.9955': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1633 raised an error (recorded in the `errors` attribute of the result table): '18:44:41.834 -3:05:51.3368': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1635 raised an error (recorded in the `errors` attribute of the result table): '18:55:06.5782 -22:42:04.0215': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1650 raised an error (recorded in the `errors` attribute of the result table): '19:13:44.8138 -2:35:35.6672': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1652 raised an error (recorded in the `errors` attribute of the result table): '19:23:50.1584 +14:33:02.7187': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1655 raised an error (recorded in the `errors` attribute of the result table): '19:45:49.1432 -14:54:22.2071': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1658 raised an error (recorded in the `errors` attribute of the result table): '19:59:52.7892 +40:48:19.9171': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1661 raised an error (recorded in the `errors` attribute of the result table): '20:13:17.8439 +38:32:05.307': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1664 raised an error (recorded in the `errors` attribute of the result table): '20:16:46.6712 +37:02:46.8466': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1667 raised an error (recorded in the `errors` attribute of the result table): '20:23:16.8874 +38:42:54.9747': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1671 raised an error (recorded in the `errors` attribute of the result table): '20:30:40.5087 +41:16:33.6842': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1673 raised an error (recorded in the `errors` attribute of the result table): '20:32:11.3639 +41:24:52.4367': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1675 raised an error (recorded in the `errors` attribute of the result table): '20:32:13.0663 +41:12:20.3939': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1681 raised an error (recorded in the `errors` attribute of the result table): '20:32:26.0016 +41:25:14.8833': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1684 raised an error (recorded in the `errors` attribute of the result table): '20:32:31.1415 +41:30:35.8194': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1688 raised an error (recorded in the `errors` attribute of the result table): '20:32:34.7583 +41:22:16.4288': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1693 raised an error (recorded in the `errors` attribute of the result table): '20:32:43.7421 +41:30:21.568': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1697 raised an error (recorded in the `errors` attribute of the result table): '20:32:52.4827 +41:05:27.3639': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1714 raised an error (recorded in the `errors` attribute of the result table): '20:33:07.3099 +41:25:18.0916': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1763 raised an error (recorded in the `errors` attribute of the result table): '20:33:24.6394 +41:08:32.3424': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1775 raised an error (recorded in the `errors` attribute of the result table): '20:33:54.9137 +41:22:48.1473': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1777 raised an error (recorded in the `errors` attribute of the result table): '20:33:55.6923 +41:23:13.4965': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1780 raised an error (recorded in the `errors` attribute of the result table): '20:33:59.1396 +41:20:54.4709': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1782 raised an error (recorded in the `errors` attribute of the result table): '20:34:23.8212 +41:14:54.9096': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1784 raised an error (recorded in the `errors` attribute of the result table): '20:34:27.7959 +41:31:15.6801': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1786 raised an error (recorded in the `errors` attribute of the result table): '20:34:28.404 +41:31:10.489': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1789 raised an error (recorded in the `errors` attribute of the result table): '20:34:35.132 +41:07:21.4291': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1791 raised an error (recorded in the `errors` attribute of the result table): '20:34:51.3103 +41:09:30.134': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1794 raised an error (recorded in the `errors` attribute of the result table): '20:35:02.3159 +41:28:16.9437': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1803 raised an error (recorded in the `errors` attribute of the result table): '21:37:30.9958 -54:40:34.2561': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1810 raised an error (recorded in the `errors` attribute of the result table): '21:56:09.402 +37:50:39.2146': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1814 raised an error (recorded in the `errors` attribute of the result table): '22:34:39.9834 -37:42:59.2004': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1831 raised an error (recorded in the `errors` attribute of the result table): '22:55:41.7853 +62:04:44.2435': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1845 raised an error (recorded in the `errors` attribute of the result table): '22:58:01.1342 -60:57:00.7752': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1847 raised an error (recorded in the `errors` attribute of the result table): '23:18:55.9727 +0:38:19.1834': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1849 raised an error (recorded in the `errors` attribute of the result table): '23:19:37.0026 +8:05:14.3266': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 1852 raised an error (recorded in the `errors` attribute of the result table): '23:41:46.6201 -51:11:18.8854': No astronomical object found :\n  warnings.warn(\"Warning: The script line number %i raised \"\n"
     ]
    }
   ],
   "source": [
    "coords = coord.SkyCoord(data_3_sample.ra, data_3_sample.dec,unit=(u.deg, u.deg), frame='icrs')\n",
    "result_table = Simbad.query_region(coords, radius=0.000277778 * u.deg); # 36 arcsec\n",
    "resdf = result_table.to_pandas()\n",
    "count_obj = resdf.groupby(resdf.OTYPE.tolist(),as_index=False).size()\n",
    "count_obj.to_csv('./data/obj/3_count_obj{}.csv'.format(n))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "armed-argument",
   "metadata": {
    "tags": [
     "outputPrepend"
    ]
   },
   "outputs": [
    {
     "output_type": "stream",
     "name": "stderr",
     "text": [
      "ribute of the result table): '22:23:44.9707 -2:06:38.4098': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3711 raised an error (recorded in the `errors` attribute of the result table): '22:24:20.2851 +65:37:20.1947': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3713 raised an error (recorded in the `errors` attribute of the result table): '22:25:20.1252 +17:10:56.2371': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3715 raised an error (recorded in the `errors` attribute of the result table): '22:25:28.998 +17:13:19.7744': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3717 raised an error (recorded in the `errors` attribute of the result table): '22:25:36.9755 +65:29:36.9406': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3720 raised an error (recorded in the `errors` attribute of the result table): '22:28:04.9229 +61:18:12.366': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3725 raised an error (recorded in the `errors` attribute of the result table): '22:35:37.7682 +34:01:43.6339': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3733 raised an error (recorded in the `errors` attribute of the result table): '22:42:37.514 +53:10:27.2482': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3735 raised an error (recorded in the `errors` attribute of the result table): '22:45:10.393 +26:33:42.5009': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3738 raised an error (recorded in the `errors` attribute of the result table): '22:49:48.5209 +11:40:23.9589': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3755 raised an error (recorded in the `errors` attribute of the result table): '22:57:39.7592 -41:10:28.5811': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3759 raised an error (recorded in the `errors` attribute of the result table): '22:58:11.7956 -34:58:45.5881': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3761 raised an error (recorded in the `errors` attribute of the result table): '22:58:32.9721 -34:41:28.7925': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3763 raised an error (recorded in the `errors` attribute of the result table): '22:59:40.1067 -35:02:43.7946': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3768 raised an error (recorded in the `errors` attribute of the result table): '23:05:50.563 +62:30:26.1572': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3770 raised an error (recorded in the `errors` attribute of the result table): '23:07:12.078 -22:36:28.6401': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3772 raised an error (recorded in the `errors` attribute of the result table): '23:08:40.4123 -2:00:12.4904': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3775 raised an error (recorded in the `errors` attribute of the result table): '23:11:50.021 +3:44:19.5198': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3781 raised an error (recorded in the `errors` attribute of the result table): '23:19:25.1307 -42:14:32.153': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3783 raised an error (recorded in the `errors` attribute of the result table): '23:23:21.0426 +58:49:30.5617': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3785 raised an error (recorded in the `errors` attribute of the result table): '23:23:21.0426 +58:49:30.5617': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3787 raised an error (recorded in the `errors` attribute of the result table): '23:23:22.5641 +58:50:20.4592': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3789 raised an error (recorded in the `errors` attribute of the result table): '23:23:23.7426 +58:50:24.0496': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3791 raised an error (recorded in the `errors` attribute of the result table): '23:23:24.6627 +58:49:26.7481': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3793 raised an error (recorded in the `errors` attribute of the result table): '23:23:24.6627 +58:49:26.7481': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3795 raised an error (recorded in the `errors` attribute of the result table): '23:23:24.7955 +58:50:25.9225': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3797 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3799 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3801 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3803 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3805 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3819 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3821 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3823 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3825 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3827 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3829 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3831 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3833 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3835 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3837 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3839 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3841 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3843 raised an error (recorded in the `errors` attribute of the result table): '23:23:28.8784 +58:49:43.07': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3845 raised an error (recorded in the `errors` attribute of the result table): '23:23:31.5059 +58:47:46.1205': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3847 raised an error (recorded in the `errors` attribute of the result table): '23:23:31.5059 +58:47:46.1205': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3849 raised an error (recorded in the `errors` attribute of the result table): '23:23:31.5059 +58:47:46.1205': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3851 raised an error (recorded in the `errors` attribute of the result table): '23:23:31.5059 +58:47:46.1205': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3853 raised an error (recorded in the `errors` attribute of the result table): '23:23:31.5059 +58:47:46.1205': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3855 raised an error (recorded in the `errors` attribute of the result table): '23:23:33.8973 +58:50:05.4209': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3857 raised an error (recorded in the `errors` attribute of the result table): '23:23:33.8973 +58:50:05.4209': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3859 raised an error (recorded in the `errors` attribute of the result table): '23:23:33.8973 +58:50:05.4209': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3861 raised an error (recorded in the `errors` attribute of the result table): '23:23:38.5775 +58:48:17.8538': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3863 raised an error (recorded in the `errors` attribute of the result table): '23:24:50.9259 -11:54:10.1274': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3865 raised an error (recorded in the `errors` attribute of the result table): '23:27:17.9038 +26:46:44.0543': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3868 raised an error (recorded in the `errors` attribute of the result table): '23:27:41.9426 +23:35:13.0657': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3870 raised an error (recorded in the `errors` attribute of the result table): '23:30:56.359 +39:27:33.3524': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3874 raised an error (recorded in the `errors` attribute of the result table): '23:35:30.2502 -36:03:53.7185': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3877 raised an error (recorded in the `errors` attribute of the result table): '23:40:26.5668 -12:30:19.9543': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 3887 raised an error (recorded in the `errors` attribute of the result table): '23:59:15.0766 +62:26:39.3853': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\n"
     ]
    }
   ],
   "source": [
    "coords = coord.SkyCoord(data_4_sample.ra, data_4_sample.dec,unit=(u.deg, u.deg), frame='icrs')\n",
    "result_table = Simbad.query_region(coords, radius=0.000277778 * u.deg); # 36 arcsec\n",
    "resdf = result_table.to_pandas()\n",
    "count_obj = resdf.groupby(resdf.OTYPE.tolist(),as_index=False).size()\n",
    "count_obj.to_csv('./data/obj/4_count_obj{}.csv'.format(n))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "wireless-combat",
   "metadata": {
    "tags": [
     "outputPrepend"
    ]
   },
   "outputs": [
    {
     "output_type": "stream",
     "name": "stderr",
     "text": [
      "esult table): '22:19:28.3505 -27:27:28.2438': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5656 raised an error (recorded in the `errors` attribute of the result table): '22:23:36.6944 -2:05:11.6266': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5658 raised an error (recorded in the `errors` attribute of the result table): '22:26:14.5673 +17:32:03.4184': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5661 raised an error (recorded in the `errors` attribute of the result table): '22:28:34.7859 +20:35:38.9267': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5668 raised an error (recorded in the `errors` attribute of the result table): '22:34:03.3826 -53:50:11.7716': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5675 raised an error (recorded in the `errors` attribute of the result table): '22:36:21.7866 -25:57:43.5506': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5677 raised an error (recorded in the `errors` attribute of the result table): '22:36:53.5887 -45:59:37.4295': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5684 raised an error (recorded in the `errors` attribute of the result table): '22:39:12.5988 -5:43:23.1467': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5686 raised an error (recorded in the `errors` attribute of the result table): '22:39:26.7516 -17:19:40.4313': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5698 raised an error (recorded in the `errors` attribute of the result table): '22:41:18.6509 +53:08:51.1674': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5702 raised an error (recorded in the `errors` attribute of the result table): '22:48:49.7511 -15:57:48.4081': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5709 raised an error (recorded in the `errors` attribute of the result table): '22:54:01.1331 +62:45:30.4678': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5721 raised an error (recorded in the `errors` attribute of the result table): '22:56:47.2041 -43:24:39.4643': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5726 raised an error (recorded in the `errors` attribute of the result table): '22:57:25.8263 -61:03:48.6162': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5728 raised an error (recorded in the `errors` attribute of the result table): '22:57:45.1835 -36:37:23.4664': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5730 raised an error (recorded in the `errors` attribute of the result table): '22:58:03.9673 +5:28:46.0931': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5734 raised an error (recorded in the `errors` attribute of the result table): '22:59:15.6123 -40:40:36.568': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5736 raised an error (recorded in the `errors` attribute of the result table): '23:00:57.3582 +30:05:02.8066': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5738 raised an error (recorded in the `errors` attribute of the result table): '23:01:50.1972 -38:47:20.172': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5740 raised an error (recorded in the `errors` attribute of the result table): '23:02:22.3075 +8:50:22.4594': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5742 raised an error (recorded in the `errors` attribute of the result table): '23:02:32.6476 -40:29:36.078': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5745 raised an error (recorded in the `errors` attribute of the result table): '23:03:01.1669 +55:17:57.9526': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5752 raised an error (recorded in the `errors` attribute of the result table): '23:07:44.1393 -9:53:58.1096': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5754 raised an error (recorded in the `errors` attribute of the result table): '23:07:56.6173 -22:51:46.6439': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5756 raised an error (recorded in the `errors` attribute of the result table): '23:08:17.417 -1:55:52.631': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5758 raised an error (recorded in the `errors` attribute of the result table): '23:11:54.1272 -21:46:44.8222': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5763 raised an error (recorded in the `errors` attribute of the result table): '23:17:24.4591 +42:56:34.4457': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5765 raised an error (recorded in the `errors` attribute of the result table): '23:17:46.6322 +0:24:55.31': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5768 raised an error (recorded in the `errors` attribute of the result table): '23:18:58.1015 +0:46:15.2625': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5775 raised an error (recorded in the `errors` attribute of the result table): '23:21:41.8395 -2:15:22.1747': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5777 raised an error (recorded in the `errors` attribute of the result table): '23:21:55.582 -2:18:22.8939': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5779 raised an error (recorded in the `errors` attribute of the result table): '23:23:25.8048 +58:50:04.3574': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5781 raised an error (recorded in the `errors` attribute of the result table): '23:23:54.8003 +58:50:00.5392': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5785 raised an error (recorded in the `errors` attribute of the result table): '23:25:36.8491 +42:20:03.3385': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5787 raised an error (recorded in the `errors` attribute of the result table): '23:25:48.4718 +26:38:27.5129': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5790 raised an error (recorded in the `errors` attribute of the result table): '23:26:06.7073 +42:22:06.354': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5801 raised an error (recorded in the `errors` attribute of the result table): '23:34:27.5596 -36:11:04.6313': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5803 raised an error (recorded in the `errors` attribute of the result table): '23:34:39.3828 -36:01:38.4336': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5805 raised an error (recorded in the `errors` attribute of the result table): '23:35:07.0849 -85:10:15.0733': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5807 raised an error (recorded in the `errors` attribute of the result table): '23:35:41.2963 -36:08:17.5074': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5809 raised an error (recorded in the `errors` attribute of the result table): '23:35:59.9579 -45:55:55.1293': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5811 raised an error (recorded in the `errors` attribute of the result table): '23:36:09.8038 +21:20:25.2357': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5813 raised an error (recorded in the `errors` attribute of the result table): '23:36:30.4843 +21:08:47.4397': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5816 raised an error (recorded in the `errors` attribute of the result table): '23:37:09.5645 +20:56:14.3294': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5818 raised an error (recorded in the `errors` attribute of the result table): '23:37:41.1863 +27:08:05.0383': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5825 raised an error (recorded in the `errors` attribute of the result table): '23:41:56.8889 -54:17:44.3949': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5827 raised an error (recorded in the `errors` attribute of the result table): '23:43:15.5499 -55:23:22.7692': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5829 raised an error (recorded in the `errors` attribute of the result table): '23:43:35.9424 -15:19:16.2326': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5832 raised an error (recorded in the `errors` attribute of the result table): '23:44:40.4878 -64:04:29.5824': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5836 raised an error (recorded in the `errors` attribute of the result table): '23:45:18.8502 +9:16:19.8358': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5838 raised an error (recorded in the `errors` attribute of the result table): '23:47:45.0209 -28:08:25.7804': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5844 raised an error (recorded in the `errors` attribute of the result table): '23:54:47.0035 -50:55:38.4742': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5846 raised an error (recorded in the `errors` attribute of the result table): '23:56:15.951 -34:41:27.9432': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5848 raised an error (recorded in the `errors` attribute of the result table): '23:56:49.9585 -35:00:36.1874': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5851 raised an error (recorded in the `errors` attribute of the result table): '23:57:51.1427 -32:42:56.3084': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5854 raised an error (recorded in the `errors` attribute of the result table): '23:58:06.7171 -32:37:57.0563': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5856 raised an error (recorded in the `errors` attribute of the result table): '23:58:08.826 -32:34:03.6433': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5858 raised an error (recorded in the `errors` attribute of the result table): '23:58:32.6254 -32:38:19.2939': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5860 raised an error (recorded in the `errors` attribute of the result table): '23:58:53.3908 -60:55:34.852': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\nC:\\Users\\samue\\miniconda3\\envs\\astrox\\lib\\site-packages\\astroquery\\simbad\\core.py:135: UserWarning: Warning: The script line number 5862 raised an error (recorded in the `errors` attribute of the result table): '23:59:25.6994 -50:07:54.3791': No astronomical object found :  \n  warnings.warn(\"Warning: The script line number %i raised \"\n"
     ]
    }
   ],
   "source": [
    "coords = coord.SkyCoord(data_5_sample.ra, data_5_sample.dec,unit=(u.deg, u.deg), frame='icrs')\n",
    "result_table = Simbad.query_region(coords, radius=0.000277778 * u.deg); # 36 arcsec\n",
    "resdf = result_table.to_pandas()\n",
    "count_obj = resdf.groupby(resdf.OTYPE.tolist(),as_index=False).size()\n",
    "count_obj.to_csv('./data/obj/5_count_obj{}.csv'.format(n))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}